pkg/controller/wip/controller.go

```diff
@@ -382,16 +387,30 @@ func (c *controller) reconcileInstance(instance *v1alpha1.Instance) {
 	glog.V(4).Infof("Creating client for Broker %v, URL: %v", broker.Name, broker.Spec.URL)
 	brokerClient := c.brokerClientCreateFunc(broker.Name, broker.Spec.URL, username, password)
 
-	// TODO: parameters
+	parameters, err := unmarshalParameters(instance.Spec.Parameters.Raw)
```

Comment:
I realized last night while doing some cleanups to the API that the parameters field should really be a pointer so that we can easily test for 'unset'.  I'll clean this up in a followup today after this is merged.